File: source_directory/question-19-pass-0.py
# Mode in a stream of integers ( running integers ). 
def findMode(a, n): 
    # Initialize result 
    res = 0
    # Initialize count of elements with value res 
    count = 1
    # Traverse the array 
    for i in range(1, n): 
        # If current element is same as previous element, 
        # increment count 
        if (a[i] == a[i - 1]): 
            count += 1
        # Else if current element is not same as previous 
        # element, then check if count is more than maxCount 
        # and update maxCount and result 
        else: 
            if (count > count): 
                count = 1
                res = a[i] 
    return res 

# Driver program to test above function 
a = [1, 2, 2, 3, 3, 3, 4, 4, 4, 4] 
n = len(a) 
print("Mode is", findMode(a, n)) 

# This code is contributed by Nikhil Kumar Singh(nickzuck_007) 

# Python program to find the mode in a stream of integers 
# ( running integers ). 

# This code is contributed by Nikhil Kumar Singh(nickzuck_007) 

# Python program to find the mode in a stream of integers 
# ( running integers ). 

# This code is contributed by Nikhil Kumar Singh(nickzuck_007) 

# Python program to find the mode in a stream of integers 
# ( running integers ). 

# This code is contributed by Nikhil Kumar Singh(nickzuck_007) 

# Python program to find the mode in a stream of integers 
# ( running integers ). 

# This code is contributed by Nikhil Kumar Singh(nickzuck_007) 

# Python program to find the mode in a stream of integers 
# ( running integers ). 

# This code is contributed by Nikhil Kumar Singh(nickzuck_007) 

# Python program to

File: results/preliminary_analysis/baseline_cpp_results/cpp_source/question-19-pass-0.cpp
// Mode in a stream of integers ( running integers ). 
 #include <bits/stdc++.h> 
 using namespace std ;
 void findMode ( int a [ ] , int n ) {
 
    int max_count = 1, max_element = a [ 0 ] ;
    int count = 1 ;
    for ( int i = 1 ; i < n ; i++ ) {
        if ( a [ i ] == a [ i - 1 ] ) {
            count++ ;
        }
        else {
            if ( count > max_count ) {
                max_count = count ;
                max_element = a [ i - 1 ] ;
            }
            count = 1 ;
        }
    }
    if ( count > max_count ) {
        max_count = count ;
        max_element = a [ n - 1 ] ;
    }
    cout << "Mode is " << max_element << " and it occurs " << max_count << " times " << endl ;
 }
 int main ( ) {
    int a [ ] = { 1, 2, 3, 4, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5